python/tidy/tidy.py

```diff
@@ -128,6 +133,16 @@ def is_iter_empty(iterator):
         return False, iterator
 
 
+def normalize_path(path: str) -> str:
+    abs_path = os.path.abspath(path)
+
+    if abs_path.startswith(TOPDIR):
+        relative = abs_path[len(TOPDIR) :]
+        return f".{relative}"
+
+    return path
+
```

Comment:
Can you replace this with a one line call to [`relpath`](https://docs.python.org/3/library/os.path.html#os.path.relpath)?

```python
    os.path.relpath(os.path.abspath(path), TOPDIR))
```